Makefile

```diff
@@ -100,4 +99,9 @@ test_with_coverage:
 	exit $$exit_code
 
 merge_coverage_results:
-	cargo llvm-cov report --summary-only --json --output-path ${UNIT_COVERAGE_JSON}
\ No newline at end of file
+	cargo llvm-cov report --summary-only --json --output-path ${UNIT_COVERAGE_JSON}
+
+.ONESHELL:
```

Comment:
nit: more of a note to @Trinaa and @jrconlin as we have this in autopush too -- it seems `.ONESHELL` is a Makefile wide [special target](https://www.gnu.org/software/make/manual/html_node/Special-Targets.html) that only needs to be [set once](https://www.gnu.org/software/make/manual/html_node/One-Shell.html), _"If the .ONESHELL special target appears anywhere in the makefile then all recipe lines for each target.."_